src/test/scala/com/ubirch/discovery/models/StoreSpec.scala

```diff
@@ -8,13 +8,17 @@ import com.ubirch.discovery.services.connector.GremlinConnector
 import com.ubirch.discovery.util.RemoteJanusGraph
 import org.scalatest.{ Assertion, Ignore }
 
+/*
+ * This store part is currently included in AbstractDiscoveryService.class, so can't be tested individually.
```

Comment:
I added this comment because this spec seems not be necessary, but can't decide how to treat these cases